package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "firestore-jest-mock",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Would you remind removing this change? We'll take care of the release version bump

Reply:
Done, and committed.